test/octree/test_octree_iterator.cpp

```diff
@@ -258,16 +258,6 @@ TEST_F (OctreeBaseBeginEndIteratorsTest, End)
   EXPECT_EQ (it_a_1, it_a_2);
   EXPECT_NE (it_a_1, it_b);
   EXPECT_NE (it_a_2, it_b);
-
-  // Different max depths are not the same iterators
-  IteratorT it_m = oct_a_.end ();
-  IteratorT it_m_1 = oct_a_.end (1);
-  IteratorT it_m_2 = oct_a_.end (2);
-  IteratorT it_m_b_1 = oct_b_.end (1);
-
-  EXPECT_NE (it_m_1, it_m_2);
-  EXPECT_EQ (it_m_2, it_m); // tree depth is 2
-  EXPECT_NE (it_m_1, it_m_b_1);
```

Comment:
You should leave the test in which you verify that the end iterators from different octrees are not the same.

Reply:
These tests concern iterators from different depth and differnet octree. The test about iterators from different octrees is preserved above.